File: Python/array_list.py
class ArrayList:
    def _init_(self):
        self.arr = []

    def insert_end(self, value):
        self.arr.append(value)   # dynamic array growth handled by Python internally

    def get_size(self):
        return len(self.arr)

    def print_list(self):
        for item in self.arr:
            print(item, end="  ")
        print()

File: C/arraylist.c
#include <stdio.h>
#include "arraylist.h"

void insertEnd(ArrayList* list, int value) {
    list->arr[list->size++] = value;
}

int getSizeArray(ArrayList* list) {
    return list->size;
}

void printArray(ArrayList* list) {
    for (int i = 0; i < list->size; i++) {
        printf("%d ", list->arr[i]);
    }
    printf("\n");
}